guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,35 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<p>This section looks like an overview of Transformers, it is not conveying any information about how Transformers can be used for Entity Recognition. Either the header needs to be modified to something like "Transformers Overview" or the content needs to be modified to show how transformers are used for entity recognition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Providing details of how transformers are used for entity recognition will make the content lengthy and too technical for the guide purpose (because then we have to explain concepts like how transformer tokenizer works including things like word piece tokenization, then for entity recognition explain what is token classification and how it can be used for entity recognition). 

I will be modifying the heading like this

EntityRecognizer Architecture
- Transformers Overview
- Entity recognition with spaCy